conans/client/migrations_settings.py

```diff
@@ -2413,6 +2414,12 @@
                 <<: *visual_studio
             apple-clang:
                 <<: *apple_clang
+    intel-cc:
+        version: ["2021.1", "2021.2", "2021.3"]
+            update: [None, ANY]
+        mode: [None, "classic", "dpcpp"]
```

Comment:
what is mode `None` in this case? is it ICX?
we have 3 compiler:
```
        - ICX - Intel oneAPI DPC++/C++ Compiler
        - ICC Classic - Intel C++ Compiler Classic
        - DPCPP - Intel oneAPI DPC++/C++ Compiler
 ```
if so, how about explicit ["icx", "classic", "dpcpp"]?